backend/src/index.js

```diff
@@ -0,0 +1,32 @@
+require('dotenv').config();
+const express = require('express');
+const { Client } = require('pg');
+const app = express();
+
+const client = new Client({
+  connectionString: process.env.DATABASE_URL,
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+client.connect()
+  .then(() => console.log('Connected to the database'))
+  .catch(err => console.error('Connection error', err.stack));
+
+app.get('/items', async (req, res) => {
+  try {
+    const result = await client.query('SELECT * FROM items');
+    res.json(result.rows);
+  } catch (err) {
+    console.error(err);
+    res.status(500).send('Server error');
+  }
+});
+
+const PORT = process.env.PORT || 3000;
+app.listen(PORT, () => {
+  console.log(`Server running on port ${PORT}`);
+});
+
+module.exports = client;
```

Comment:
if we change to pool

```suggestion
module.exports = pool;
```